nautobot_secrets_providers/providers/delinea.py

```diff
@@ -30,34 +30,34 @@
             SecretServerError,
         )
 
-        thycotic_installed = True  # pylint: disable=invalid-name
+        delinea_installed = True  # pylint: disable=invalid-name
     except ImportError:
-        thycotic_installed = False  # pylint: disable=invalid-name
+        delinea_installed = False  # pylint: disable=invalid-name
 
 from nautobot.core.forms import BootstrapMixin
 from nautobot.extras.secrets import exceptions, SecretsProvider
 
-from .choices import ThycoticSecretChoices
+from .choices import DelineaSecretChoices
 
 __all__ = (
-    "ThycoticSecretServerSecretsProviderId",
-    "ThycoticSecretServerSecretsProviderPath",
+    "DelineaSecretServerSecretsProviderId",
+    "DelineaSecretServerSecretsProviderPath",
 )
 
 
-class ThycoticSecretServerSecretsProviderBase(SecretsProvider):
-    """A secrets provider for Thycotic Secret Server."""
+class DelineaSecretServerSecretsProviderBase(SecretsProvider):
+    """A secrets provider for Delinea Secret Server."""
```

Comment:
I'm slightly concerned about the possibility for this to be a breaking change if anyone has their own App that directly uses or extends this class. I think it's a low risk, but something to consider nonetheless.

Reply:
Yea - probably just pull the bandaid off and have this all included in a major version release